js/viz/core/default.js

```diff
@@ -1457,6 +1457,94 @@ registerTheme({
                 alignment: RIGHT
             }
         }
+    },
+    funnel: {
+        sortData: true,
+        valueField: "value",
+        colorField: "color",
+        argumentField: "argument",
```

Comment:
Why default argumentfield and valueField values are differ from similar ones in dxChart?